sdk/maps/Azure.ResourceManager.Maps/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.1.0-beta.4 (Unreleased)
+## 1.1.0 (Unreleased)
```

Comment:
Please specify the release date for this new release

Reply:
Done. Thanks for the suggestion.